homeassistant/components/template/light.py

```diff
@@ -415,7 +418,10 @@ def update_temperature(self):
         if self._temperature_template is None:
             return
         try:
-            temperature = int(self._temperature_template.async_render())
+            render = self._temperature_template.async_render()
+            if render in ("None", ""):
+                return
```

Comment:
Why not set `_temperature` to None like you did to brightness? It will keep the previous value otherwise.

Reply:
Fair point. Done.